scripts/scil_surface_create.py

```diff
@@ -0,0 +1,205 @@
+
+    p.add_argument('out_surface',
+                   help='Output surface (.vtk)')
+
+    g2 = p.add_argument_group("Options for labels input")
+    mxg2 = g2.add_mutually_exclusive_group()
+    mxg2.add_argument('--list_indices', nargs='+',
+                      help='List of labels indices to use for the surface.')
+    mxg2.add_argument('--each_index', action='store_true',
+                      help='Create a surface per index. It will use the '
+                           'out_surface basename to create the output files.')
+
+    g3 = p.add_argument_group('Options for volume input')
+    g3.add_argument('--value', default=0.5,
+                    type=ranged_type(float, 0, None, min_excluded=False),
+                    help='Isosurface threshold value used. '
+                         'This value is called isovalue in mbcube.\n'
+                         'Example: For a binary mask (with 0 and 1), '
+                         '0.5 will generate a surface in the middle '
+                         'of the transition. [%(default)s]')
+
+    morpho_g = p.add_argument_group('Morphology options')
+    morpho_g.add_argument('--smooth',
+                          type=ranged_type(float, 0, None, min_excluded=False),
+                          help='Smoothing size with'
+                               ' 1 implicit step. [%(default)s]')
+    morpho_g.add_argument('--erosion',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Erosion: number of iterations. [%(default)s]')
+    morpho_g.add_argument('--dilation',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Dilation: number of iterations. [%(default)s]')
+    morpho_g.add_argument('--opening',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Opening (dilation of the erosion): number '
+                               'of iterations. [%(default)s]')
+    morpho_g.add_argument('--closing',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Closing (erosion of the dilation): number '
+                               'of iterations. [%(default)s]')
+
+    p.add_argument('--fill', action='store_true',
+                   help='Fill holes in the image. [%(default)s]')
+
+    p.add_argument('--vtk2vox', action='store_true',
+                   help='Do not transform to vox2vtk. [%(default)s]')
+
+    add_verbose_arg(p)
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+    logging.getLogger().setLevel(logging.getLevelName(args.verbose))
+
+    assert_inputs_exist(parser, [], [args.in_labels, args.in_mask])
+    assert_outputs_exist(parser, args, args.out_surface)
+
+    masks = []
+
+    if args.in_labels:
+        # Default value for isosurface
+        args.value = 0.5
```

Comment:
see previous comment, otherwise  # 129, and # 148 could be grouped before the if